### DISCUSSION TITLE
Set the actual Airflow version to Airflow OpenAPI definition

### DISCUSSION THREAD

**Discussion by igr** [apache/airflow#25127] [General] [answered]
The [API](https://airflow.apache.org/docs/apache-airflow/stable/stable-rest-api-ref.html) shows version `1.0.0`. This OpenAPI version should match the version of the Airflow.

Having a proper API version helps with releasing of the generated API clients (like [this](https://github.com/apache/airflow-client-go/issues/25)).

**Comment by potiuk** [comment, marked as answer]
Again. Devlist discussion is wher you shoudl start. Similar to this one (which was far from endless). https://lists.apache.org/thread/8yd7o9wsj7jgcwvpzz9pcc74gt1l7r65 - please bring it to the devlist, this is the only place where it should be discussed - it's far too important to be in an issue or Github Discussion.

**Comment by potiuk** [comment]
Also just to explain WHY. Devlist is the only place where important things shoudl be discussed which have big impact on the project. By discussing it in Github Discussion, you are leaving out people who should be involved and they missed it among the myriad of others issues and discussion. That's why we have devlist, where really important things are discussed. We are highly -distributed and diverse team of people and the devlist is the only place where decisions like that should be made.

* "If it did not happen on the Devlist, it did not happen" is one of the mantras of the Apache Software Foundation
